python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -128,7 +128,12 @@ def from_pretrained(cls,
         """
         pretrained_model_name_or_path = kwargs.get("pretrained_model_name_or_path", None) \
             if len(args) == 0 else args[0]
-        config_dict, _ = PretrainedConfig.get_config_dict(pretrained_model_name_or_path)
+        try:
```

Comment:
add a parameter `model_hub`?

Reply:
> add a parameter `model_hub`?

Sure, added a new parameter `model_hub` to specify model hub, options are 'huggingface' and 'modelscope', default to be 'huggingface'.